app/models/pod_version.rb

```diff
@@ -61,6 +61,13 @@ def public_attributes
         { 'name' => name, 'created_at' => created_at }
       end
 
+      # Where should it go in the current state of the repo
+      def current_destination_path
+        File.join('Specs',
```

Comment:
this method could just be `self.class.destination_path(pod.name, name, DateTime.now)`